src/elisp/treemacs-interface.el

```diff
@@ -251,7 +251,7 @@ Stay in the current window with a single prefix argument ARG, or close the
 treemacs window with a double prefix argument."
   (interactive "P")
   (treemacs--execute-button-action
-   :window (aw-select "Select window")
+   :window (ace-window "Select window")
```

Comment:
`ace-window` just calls `ace-select-window`, just without prefix arg options we don't need. Might as well use that.

Reply:
This commit message should be prefixed with `[Interface]`.